i2c/hw/sample-implementations/Atmel_SAMD2_series/sensirion_hw_i2c_implementation.c

```diff
@@ -52,12 +52,12 @@ void sensirion_i2c_init(void) {
 }
```

Comment:
No return at end of function. These changes are hard to test if these sample implementations still work.